core/src/main/java/hudson/Functions.java

```diff
@@ -25,6 +25,7 @@
  */
 package hudson;
 
+import com.google.inject.internal.asm.$ByteVector;
```

Comment:
This doesn't look like it belongs here.

Reply:
Thanks. I don't know how it got there. I removed it. Thanks for pointing that out.